### PR TITLE
HW12T01

### PR DESCRIPTION
**Pull request by LinaKrevetochka**

### CODE REVIEW

File: hw/homework12/example/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (Cjkjvfnby) [inline hw/homework12/example/admin.py]: I don't see any models. Is this file required?

File: hw/homework12/example/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("last_name", models.CharField(max_length=20)),
```
Comment (Cjkjvfnby) [inline hw/homework12/example/migrations/0001_initial.py]: Ostap-Suleyman-Berta-Maria-Bender-Bey would not approve it.

File: hw/homework12/example/tests.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (Cjkjvfnby) [inline hw/homework12/example/tests.py]: Do you need this file?

File: hw/homework12/homework12/wsgi.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+"""
+WSGI config for homework12 project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/howto/deployment/wsgi/
```
Comment (Cjkjvfnby) [inline hw/homework12/homework12/wsgi.py]: You have asgi and wsgi.  Do you really need both of them?

File: requirements-dev.txt [hunk 5]
```diff
@@ -27,4 +31,5 @@ BeautifulSoup4
 
 requests
 
-lxml
\ No newline at end of file
+lxml
```
Comment (Cjkjvfnby) [inline requirements-dev.txt:34]: Is it still required?

**Review by Cjkjvfnby — COMMENTED** [review]
Please remove "empty" files and cleanup dependencies.

File: requirements-dev.txt (hunk 5 above)
Comment (LinaKrevetochka) [reply requirements-dev.txt:34]: We need it because pytest always runs all tests for all homeworks.

**Review by Cjkjvfnby — APPROVED** [review]
